### PR TITLE
[Api.ai] sessionId parameter

### PR DESCRIPTION
**Pull request by ribb3r**
Keep continue the Api.ai course of conversation using the sessionId parameter

### CODE REVIEW

**Comment by christophrumpel** [comment]
Guess the error occurs because of the driver method `getName` you are calling now that wasn't there before and this method call wasn't expected.

**Comment by ribb3r** [comment]
Thanks @christophrumpel! I realized that there is an existing function `getOriginatedConversationIdentifier` I used this to keep the user in the conversation flow.

**Comment by mpociot** [comment]
Thank you!

File: src/Mpociot/BotMan/Middleware/ApiAi.php [hunk 1]
```diff
@@ -65,7 +65,7 @@ public function handle(Message &$message, DriverInterface $driver)
     {
         $response = $this->http->post($this->apiUrl, [], [
             'query' => [$message->getMessage()],
-            'sessionId' => substr($message->getOriginatedConversationIdentifier(), 0, 36),
```
Comment (asterion) [inline src/Mpociot/BotMan/Middleware/ApiAi.php]: Thanks :)

**Pull request merged**